spec/models/supplejack_api/user_set_spec.rb

```diff
@@ -77,12 +77,11 @@ module SupplejackApi
           expect(user_set.subjects).to eq ["Dogs", "Cats"]
         end
 
-        # Suspended till subject to tag syn is removed
-        # it "removes html tags from the tags" do
-        #   user_set.tags = ["Dogs", "<b>Cats</b>"]
-        #   user_set.strip_html_tags
-        #   expect(user_set.tags).to eq ["Dogs", "Cats"]
-        # end
+        it "removes html tags from the tags" do
+          user_set.tags = ["Dogs", "<b>Cats</b>"]
+          user_set.strip_html_tags
```

Comment:
This is kinda out of the scope of this code review, but since `strip_html_tags` mutates it's instance object don't you think it would be good to add a banged version of it `strip_html_tags!` instead so it's explicit says that it mutates the object?